shared/AppInsightsCommon/src/Util.ts

```diff
@@ -335,24 +341,85 @@ export class Util {
         return CoreUtils._canUseCookies && Util.document && Util.document.cookie;
     }
 
+    public static disallowsSameSiteNone(userAgent:string) {
+        if (!_isString(userAgent)) {
+            return false;
+        }
+    
+        // Cover all iOS based browsers here. This includes:
+        // - Safari on iOS 12 for iPhone, iPod Touch, iPad
+        // - WkWebview on iOS 12 for iPhone, iPod Touch, iPad
+        // - Chrome on iOS 12 for iPhone, iPod Touch, iPad
+        // All of which are broken by SameSite=None, because they use the iOS networking stack
+        if (userAgent.indexOf("CPU iPhone OS 12") !== -1 || userAgent.indexOf("iPad; CPU OS 12") !== -1) {
```

Comment:
Chromium page recommend more elaborated regex to identify these, are we sure this will be enough?

Reply:
Yes, these are simplified versions based on actual usages.